nix/pkgs.nix

```diff
@@ -25,9 +25,19 @@ let
 
   # Override some packages and utilities
   pkgsOverlay = import ./overlay.nix;
+
+  # Fix for lack of Android SDK for M1 Macs.
+  systemOverride = let
+    inherit (builtins) currentSystem getEnv;
+  in
+    if getEnv "CI" != "true" && currentSystem == "aarch64-darwin" then
```

Comment:
can we add another flag here for folks that do not want intel packages by default? so that everyone gets intel packages by default and do not face issues while building Android on MacOs but for the few of us that want faster builds locally we could override it?

Reply:
You know you can just set `CI=true` :D. Or do you want one with a name that's more explicit?

Reply:
`CI=true` works :D Thanks @jakubgs